src/main/flight/pid.h

```diff
@@ -85,11 +86,6 @@ typedef struct pidProfile_s {
     uint16_t rateAccelLimit;                // accel limiter roll/pitch deg/sec/ms
 } pidProfile_t;
 
-#if FLASH_SIZE <= 128
```

Comment:
Sorry, I didn't spot that, I thought it had just been removed.

Can you rebase?